pkg/opa/opa_calls.go

```diff
@@ -194,7 +195,8 @@ func SyncOPAChecks(syncDir, org, insightsToken, host string, fullsync, dryrun bo
 
 func getHeaders(token string) req.Header {
 	return req.Header{
-		"Authorization": fmt.Sprintf("Bearer %s", token),
-		"Accept":        "application/json",
+		"X-FAIRWINDS-CLI-VERSION": cliversion.GetVersion(),
```

Comment:
For consistency (even though case shouldn't matter for a header)
```suggestion
		"X-Fairwinds-CLI-Version": cliversion.GetVersion(),
```